src/skeleplex/graph/skeleton_graph.py

```diff
@@ -50,6 +50,52 @@ def skeleton_graph_decoder(json_object):
     return json_object
 
 
+def make_graph_directed(graph: nx.Graph, origin=int) -> nx.DiGraph:
+    """Return a directed graph from an undirected graph.
+
+    The directed graph has the same nodes and edges as the undirected graph.
+
+    Parameters
+    ----------
+    graph : nx.Graph
+        The undirected graph to convert to a directed graph.
+    origin : int
+        The node to use as the origin node for the directed graph.
+        The origin node will have no incoming edges.
+    """
+    if isinstance(graph, nx.DiGraph):
+        print("The input graph is already a directed graph.")
+        return graph
+    if len(list(nx.connected_components(graph))) > 1:
+        Warning("""
```

Comment:
Can you please use warnings.warn instead? I don't think the `Warning` class will actually emit the warning by itself.

https://docs.python.org/3/library/warnings.html#warnings.warn